src/cli.cpp

```diff
@@ -230,6 +242,28 @@ void CLI::printKey(char *pk) {
                spk.substr(20,5));
 }
 
+int CLI::stripKey(const char *in_key, char *out_key) {
```

Comment:
It's possible to explicitly state we're accepting 29-character keys and releasing 25-character stripped keys.
Could use references `(&out_key)[25]`. Not sure though if it'll fit, we'll probably move to std::string. Just a suggestion

Reply:
tentatively added a PK_LENGTH specifier on out_key, however I wouldn't be sure on specifying input buffer sizes for the input because it can vary - as-is the function can accept both 25 char (no dashes) and 29 char (with dashes) keys.